app/javascript/lib/commons/index.js

```diff
@@ -30,4 +31,9 @@ document.addEventListener("DOMContentLoaded", () => {
   if (!disableTurbolinks) {
     Turbolinks.start()
   }
-})
\ No newline at end of file
+})
+
+$(document).on("turbolinks:load", () => {
```

Comment:
Hum, but this is something I already let you know: those pages whose turbolinks are disabled, they won't never reach this code.
How do you deal with that?